lampo-common/src/keys.rs

```diff
@@ -1,8 +1,10 @@
 use std::{sync::Arc, time::SystemTime};
 
-use bitcoin::secp256k1::{Secp256k1, SecretKey};
 use lightning::sign::{InMemorySigner, NodeSigner, OutputSpender, SignerProvider};
+use lightning_invoice::RawBolt11Invoice;
 
+use crate::ldk::bitcoin::secp256k1::Secp256k1;
+use crate::ldk::bitcoin::secp256k1::SecretKey;
```

Comment:
we should have access to `use crate::bitcoin`

Reply:
Thanks, fixed